packages/material-ui/src/ButtonGroup/ButtonGroup.test.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { assert } from 'chai';
+import {
+  createMount,
+  createShallow,
+  createRender,
+  // describeConformance,
+  getClasses,
+} from '@material-ui/core/test-utils';
+import Button from '../Button/Button';
+import ButtonGroup from './ButtonGroup';
+
+describe('<ButtonGroup />', () => {
+  let mount;
+  let shallow;
+  let render;
+  let classes;
+
+  before(() => {
+    mount = createMount({ strict: true });
+    shallow = createShallow({ dive: true });
+    render = createRender();
+    classes = getClasses(
+      <ButtonGroup>
+        <Button>Hello World</Button>
+      </ButtonGroup>,
+    );
+  });
+
+  after(() => {
+    mount.cleanUp();
+  });
+
+  // describeConformance(<ButtonGroup><Button>Conformance?</Button></ButtonGroup>, () => ({
```

Comment:
To enable.

Reply:
I think this works fine. The problem is that a StrictMode mount is used.

Reply:
It doesn't seem to matter whether `strict` is set to `true` or `false` (what does that do BTW?). Either way it fails with:

  31 passing (1s)
  1 failing

  1) <ButtonGroup />
       Material-UI component API
         does spread props to the root component:
     TypeError: Enzyme::Selector expects a string, object, or Component Constructor
      at reduceTreeBySelector (node_modules/enzyme/build/selectors.js:508:11)
      at /Users/mbrookes/Projects/material-ui/node_modules/enzyme/build/selectors.js:515:12
      at Array.map (<anonymous>)
      at reduceTreesBySelector (node_modules/enzyme/build/selectors.js:514:23)
      at ReactWrapper.find (node_modules/enzyme/build/ReactWrapper.js:739:63)
      at find (packages/material-ui/src/test-utils/describeConformance.js:26:18)
      at Context.findRootComponent (packages/material-ui/src/test-utils/describeConformance.js:93:18)



Any suggestions?